src/components/dashboard/PotentialBarriers.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div>
-    <div class="content">
-      <p>People with fewer resources have a harder time getting vaccinated.</p>
+  <div class="content">
+    <p class="has-text-centered">
+      People with fewer resources have a harder time getting vaccinated.
+    </p>
```

Comment:
move this to the subtitle slot for the card instead of in content

Reply:
I forgot about the subtitle! I've moved it ! Let me know how it looks !